### PR TITLE
Fix alt messages for all JS docs

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/promise/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ When either of these options occur, the associated handlers queued up by a promi
 
 A promise is said to be _settled_ if it is either fulfilled or rejected, but not pending.
 
-![](promises.png)
+![A diagram showing how the Promise state transitions between pending, fulfilled, and rejected via then/catch handlers](promises.png)
```
Comment (estelle) [inline files/en-us/web/javascript/reference/global_objects/promise/index.md:35]: This diagram provides a lot of information. Is that information in the surrounding text? This image might need more content in the text and then an [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby).
>aria-labelledby and aria-describedby reference other elements to calculate a text alternative, but a label should be concise, while a description is intended to provide more verbose information; a label describes the essence of an object, while a description provides more information that the user might need.

Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/promise/index.md:35]: No—there's no text that maps to the description of this image, although the text above _kind of_ conveys the same information. Adding this to the body would make it seem a bit repetitive while there's no current text that can be used as aria-describedby. I'm not entirely sure what to do.

Comment (estelle) [inline files/en-us/web/javascript/reference/global_objects/promise/index.md:35]:
```suggestion
![Flowchart showing how the Promise state transitions between pending, fulfilled, and rejected via then/catch handlers: a pending promise is either fulfilled or rejected.  If fulfilled, the "on fulfillment" or first parameter of the then() method is executed taking asynchronous actions. If rejected, the second, on rejection, parameter or catch rejection error handling method gets executed.](promises.png)
```

Comment (estelle) [reply files/en-us/web/javascript/reference/global_objects/promise/index.md:35]: this is my attempt at "what would you tell the reader the image conveyed if the image didn't load". Feel free to edit.

Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/promise/index.md:35]:
```suggestion
![Flowchart showing how the Promise state transitions between pending, fulfilled, and rejected via then/catch handlers. A pending promise can become either fulfilled or rejected. If fulfilled, the "on fulfillment" handler, or first parameter of the then() method, is executed and carries out further asynchronous actions. If rejected, the error handler, either passed as the second parameter of the then() method or as the sole parameter of the catch() method, gets executed.](promises.png)
```

**Review by estelle — APPROVED** [review]
Thank you!

**Pull request merged**